test/performance/odp_lock_perf.c

```diff
@@ -271,6 +287,39 @@ static void print_info(test_options_t *test_options)
 	printf("\n\n");
 }
 
+static int output_summary(test_global_t *global)
+{
+	int results_size = ODPH_ARRAY_SIZE(global->results);
+	results_t res;
+
+	if (global->common_options.is_export) {
+		if (test_common_write("function name,rounds/cpu (M/s),"
+			"total rounds (M/s),cycles/cpu (per round)\n")) {
+			test_common_write_term();
+			return -1;
+		}
+	}
+
+	printf("Average results over %i threads:\n", global->test_options.num_cpu);
+	printf("%-33s %-10s %-18s %-18s %-20s\n", "function name", "duration",
+	       "rounds/cpu (M/s)", "total rounds (M/s)", "cycles/cpu (per round)");
```

Comment:
Same comments as for odp_atomic_perf regarding measuring `cycles/op` & `nsec/op` and output format.